control/joy_controller/schema/joy_controller.schema.json

```
@@ -0,0 +1,112 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Joy Controller node",
+  "type": "object",
+  "definitions": {
+    "joy_controller": {
+      "type": "object",
+      "properties": {
+        "joy_type": {
+          "type": "string",
+          "description": "Joy controller type",
+          "default": "P65",
+          "enum": ["P65", "DS4", "G29"]
+        },
+        "update_rate": {
+          "type": "number",
+          "description": "Update rate to publish control commands",
+          "default": 10.0,
+          "exclusiveMinimum": 0
+        },
+        "accel_ratio": {
+          "type": "number",
+          "description": "Ratio to calculate acceleration (commanded acceleration is ratio * operation)",
+          "default": 3.0
+        },
+        "brake_ratio": {
+          "type": "number",
+          "description": "Ratio to calculate deceleration (commanded acceleration is -ratio * operation)",
+          "default": 5.0
+        },
+        "steer_ratio": {
+          "type": "number",
+          "description": "Ratio to calculate deceleration (commanded steer is ratio * operation)",
+          "default": 0.5
+        },
+        "steering_angle_velocity": {
+          "type": "number",
+          "description": "Steering angle velocity for operation",
+          "default": 0.1
+        },
+        "accel_sensitivity": {
+          "type": "number",
+          "description": "Sensitivity to calculate acceleration for external API (commanded acceleration is pow(operation, 1 / sensitivity))",
+          "default": 1.0,
+          "exclusive": 0
```

To avoid division by 0.

---

It looks like all *_sensitivity parameters are standardized to fit within 0.001 to 1.0 in the [code](https://github.com/autowarefoundation/autoware.universe/blob/7241c83310e7d806ba1150274204d3a08859937d/control/joy_controller/src/joy_controller/joy_controller_node.cpp#L142) already, but I think it's more user friendly that we still provide information in schema file as well if we consider creating README from schema files in the future.

In that case, we might want to add maximum value defined in schema file, too.
```suggestion
          "maximum": 1.0
          "exclusiveMinimum": 0.0
```